books/free-programming-books-langs.md

```diff
@@ -1786,6 +1786,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 
 * [100 (and counting) Laravel Quick Tips](https://laraveldaily.com/wp-content/uploads/2020/04/laravel-tips-2020-04.pdf) - Povilas Korop / LaravelDaily Team (PDF)
 * [Laravel Best Practices](http://www.laravelbestpractices.com)
+* [Laravel Bootcamp](http://bootcamp.laravel.com) - Laravel Team (HTML)
```

Comment:
Looks like this is a highlevel guide which does not give more information about the framework as such. Is this a value add?

Reply:
I think this guide is more about project hands-on (building a microblogging platform) for more detailed we can see the documentation, there are 2 paths in this guide:
- using blade (beginner friendly)
- using modern js frontend stack (vue and react though inertia)

Reply:
Please move this to interactive tutorials

Reply:
moved item to interactive tutorials